docs/advanced/document-store/elasticsearch.md

```diff
@@ -118,6 +118,103 @@ da2.summary()
 
 Other functions behave the same as in-memory DocumentArray.
 
+
+### Search with `text`  
+
+Text search can be easily leveraged in a `DocumentArray` with `storage='elasticsearch'`.
+To do this text needs to be indexed using the boolean flag `'index_text'` which is set when
+the `DocumentArray` is created  with `config={'index_text': True, ...}`.  
+The following example builds a `DocumentArray` with several documents containing text and searches
+for those that have `pizza` in their text description.
+
+```python
+from docarray import DocumentArray, Document
+
+da = DocumentArray(storage='elasticsearch', config={'n_dim': 2, 'index_text': True})
+da.extend(
+    [
+        Document(id='1', text='Person eating'),
+        Document(id='2', text='Person eating pizza'),
+        Document(id='3', text='Pizza restaurant'),
```

Comment:
Same here. if this id is not required for reproducibility, i suggest we remove it. For the sake of clarity: we do not recommend manually coded id